Automattic/pocket-casts-ios
# UpNext and Profile on tabbar

Fixes #

This PR adds UpNext on the profile but keep the Profile item on the tab bar too.

**We can merge this to `release/7.64` ( we have it on the change log at the moment) or to trunk for `7.65` and updated the change log.**


| 1 | 2 |
| - | - |
| ![Simulator Screenshot - iPhone 15 Pro Max - 2024-05-14 at 17 57 27](https://github.com/Automattic/pocket-casts-ios/assets/651601/210fbfee-b173-4944-8444-6525e6380135) | ![Simulator Screenshot - iPhone 15 Pro Max - 2024-05-15 at 13 45 58](https://github.com/Automattic/pocket-casts-ios/assets/651601/65e3ad8a-7c08-423d-bd10-deabd5b8f93b) |


## To test

- Start the app
- Ensure that UpNext on the tab bar FF is enabled
- Restart the app if you needed to change the FF
- Check if UpNext and Profile show on the tab bar
- Tap on UpNext and interact with it
- Tap on Profile and check in the interaction works correctly
- Start playing any podcast that you don't have bookmarks saved
- Switch to the UpNext Tab
- Check if the episode shows correctly in the top
- Tap on the mini-player and switch to full player
- Tap on Bookmarks
- Tap on Headphone settings and check if the switch to the Profile -> Settings happens correctly
- Go to Profile and switch to another theme
- Go back to the UpNext tab and check if the theme is applied correctly

## Checklist
